### DISCUSSION TITLE
What does "benchmark_only" mean?

### DISCUSSION THREAD

**Discussion by narduzzi** [kendryte/nncase#1048] [Q&A] [answered]
Hi, it is not a bug, but just a question:

What does "benchmark_only" means?
What are the differences between turning it True or False?

For the moment I only use it with False, as default. Some models that converted with NNCASE work in the simulator, but give an "out of memory" error when deployed on K210. 
Is there a way to have the simulator behave exactly like K210?

Thanks in advance for your help

**Comment by sunnycase** [comment, marked as answer]
`benchmark_only` option generates kmodel without weights. It's only for `nncasebenchmark` program. When running `benchmark_only` kmodels, runtime dynamically generate weights using random data.
